JS2-week2/app.js

```diff
@@ -0,0 +1,72 @@
+let questions = []; // Array of all questions
+const form = document.getElementById("quiz-form");
+const questionsWrapper = document.getElementById("questions-wrapper");
+let idCounter = 1; // Increamenting each question ID
```

Comment:
You might want to start the id from 0 - it might help later on and it will be easier if it is consistent with index